app/models/creation_state.rb

```diff
@@ -18,7 +18,7 @@ def self.origins
 
   def creation_description
     if is_pos
-      pos_kind.gsub("_pos", "").humanize
+      pos_kind.to_s.gsub("_pos", "").humanize
```

Comment:
Why is this `to_s` necessary?

Reply:
`pos_kind` can be `nil` — one alternative is using the safe traversal operator on all chained method calls

Reply:
(although in the latter case the `nil` propagates)

Reply:
Ahhhh. good catch. Figured there was a reason.